docs/qml/qaoa.md

```diff
@@ -5,23 +5,25 @@ Algorithm (QAOA), first introduced by Farhi *et al.* in 2014 [^1].
 Given an arbitrary graph, the MaxCut problem consists in finding a
 graph [*cut*](https://en.wikipedia.org/wiki/Cut_(graph_theory)) which partitions
 the nodes into two disjoint sets, such that the number of edges in the
-cut is maximized. This is a very common combinatorial optimization problem which is
-computationally hard.
+cut is maximized. This is a very common combinatorial optimization problem known to be computationally hard (NP-hard).
 
-The graph used for this tutorial is a randomly generated using the `networkx` library with
-a $0.5$ probability of having an edge between two arbitrary nodes.
+The graph used for this tutorial is an unweighted graph randomly generated using the `networkx` library with
+a certain probability $p$ of having an edge between two arbitrary nodes (known as [Erdős–Rényi](https://en.wikipedia.org/wiki/Erd%C5%91s%E2%80%93R%C3%A9nyi_model) graph).
 
 ```python exec="on" source="material-block" html="1" session="qaoa"
 import numpy as np
```

Comment:
I might have missed it but I think numpy is not used.

Reply:
It is just used to set the seed with `np.random.seed(seed)`, I think it might be needed for reproducibility since `networkx` is built on top of numpy